poc-data-api/Extracting_Water_Temperature_GBR_Features.Rmd

```diff
@@ -0,0 +1,166 @@
+---
+title: "Extracting temperature using GBR features"
+author: "Denisse Fierro Arcos"
+date: "2023-06-07"
+output: 
+  github_document:
+    toc: true
+    html_preview: false
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = TRUE)
+```
+
+# Goal of this notebook
+
+This notebook will demonstrate how to use one or more Great Barrier Reef (GBR) features recognised by the Great Barrier Reef Marine Park Authority (GBRMPA) dataset to extract water temperature data from the AIMS Sea Surface Temperature Monitoring Program available at RIMReP `geoparquet` collection.
+

```

Comment:
This notebook uses a set of functions specially created for manipulating RIMReP data collections. The file `useful_spatial_functions_data_extraction.R` must exist in the same directory of this notebook

Reply:
information added